README.md

```diff
@@ -1,7 +1,7 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg-bright.png">
-  <source media="(prefers-color-scheme: light)" srcset="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
-  <img alt="Nextflow Logo" src="https://github.com/nextflow-io/trademark/blob/master/nextflow2014_no-bg.png">
+  <source media="(prefers-color-scheme: dark)" srcset="docs/_static/nextflow-logo-bg-dark.png">
```

Comment:
Great! Happier having this here, thanks.

The pedant in me would quite like a version of this image to be made with a bunch of transparent whitespace around it, as it feels a little cramped. But low priority and I can't be bothered to do it myself so don't really expect anyone else to either 😅